daml-assistant/daml-helper/src/DA/Daml/Helper/Run.hs

```diff
@@ -646,8 +646,6 @@ runNew targetFolder templateNameM pkgDeps dataDeps = do
         sdkVersion <- getSdkVersion
         let config = replace "__VERSION__"  sdkVersion
                    . replace "__PROJECT_NAME__" projectName
-                   . replace "__DEPENDENCIES__" (unlines ["  - " <> dep | dep <- pkgDeps])
-                   . replace "__DATA_DEPENDENCIES__" (unlines ["  - " <> dep | dep <- dataDeps])
```

Comment:
Hmm, I assume we removed these metavariables (`__DEPENDENCIES__`, `__DATA_DEPENDENCIES__`) from the templates?

Reply:
Even better, we removed the whole templates as part of removing `damlc migrate` :slightly_smiling_face: